website/docs/getting-started/cluster-setup/kubernetes.md

```diff
@@ -1,8 +1,111 @@
+
+## Install the Operator Lifecycle Manager(OLM)
+The Operator Lifecycle Manager(OLM) is a component of the Operator Framework, an open source toolkit to manage Kubernetes native applications, called Operators, in a streamlined and scalable way.[(Source)](https://olm.operatorframework.io/)
+
+To install operators, we will first need to install OLM [(Operator Lifecycle Manager)](https://olm.operatorframework.io/) addon to the minikube cluster.
+Running the script below will take some time to install all the necessary namespaces and other resources.
+```shell
+curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.18.3/install.sh | bash -s v0.18.3
+```
+
+To install OLM on a Kubernetes cluster setup other than minikube, please refer the [installation instructions on GitHub](https://github.com/operator-framework/operator-lifecycle-manager/#installation).
+
+## Install the Service Binding Operator
+odo uses [Service Binding Operator](https://operatorhub.io/operator/service-binding-operator) to provide the `odo link` feature which helps connect an odo component to a service or another component.
+
+Operators can be installed in a specific namespace or across the cluster(i.e. in all the namespaces). To install an operator, we need to make sure that the namespace contains `OperatorGroup` resource. Running the command below will create the necessary resource in `operators` namespace.
+```shell
+kubectl create -f https://operatorhub.io/install/service-binding-operator.yaml
+```
+
+If you want to access this resource from other namespaces as well, add your target namespace to `.spec.targetNamespaces` list.
+
+See [Verify the Operator installation](#verify-the-operator-installation) to ensure the operator is installed successfully.
+
+## Install an Operator
+1. Visit the [OperatorHub](https://operatorhub.io) website.
+2. Search for an operator of your choice.
+3. Navigate to its detail page.
+4. Click on `Install`.
+5. Follow the instruction in the installation popup.
+6. [Verify the operator was installed successfully](#verify-the-operator-installation).
+
+## Verify the Operator installation
+Wait for a few seconds for the operator to install.
+To verify that the operator is installed successfully and see the CRDs associated with it, run the following command.
+```shell
+odo catalog list services
+```
+The output can look similar to:
+```shell
+$ odo catalog list services
+Services available through Operators
+NAME                                CRDs
+datadog-operator.v0.6.0             DatadogAgent, DatadogMetric, DatadogMonitor
+service-binding-operator.v0.9.1     ServiceBinding, ServiceBinding
+```
+If you do not see your installed operator in the list, follow the [troubleshooting guide](#troubleshoot-the-operator-installation) to find the issue and debug it.
+
+## Enable Ingress addon
```

Comment:
I would move this section above "Install Operators". Enabling ingress is more important, as without it users won't be even able to access their applications.